tools/apps/src/lib.rs

```diff
@@ -744,9 +770,10 @@ impl Http3Conn {
             quiche::h3::Header::new(":status", &status.to_string()),
             quiche::h3::Header::new("server", "quiche"),
             quiche::h3::Header::new("content-length", &body.len().to_string()),
+            quiche::h3::Header::new("priority", &priority.to_string()),
```

Comment:
No, need to convert this.
```suggestion
            quiche::h3::Header::new("priority", &priority),
```